assets/i18n/en.json

```diff
@@ -111,12 +112,16 @@
     "I18N_CLASSROOM_PAGE_HEADING": "The Oppia Classroom",
     "I18N_CLASSROOM_PAGE_SEARCH_BAR_HEADING": "Explore More Lessons Made by the Community",
     "I18N_CLASSROOM_PAGE_SEARCH_BAR_SUBHEADING": "Search through our Community Library",
+    "I18N_CLASSROOM_PAGE_TITLE": "Learn <[classroomName]> with Oppia | Oppia",
```

Comment:
Isn't putting these angular markdown things into translations a bit dangerous? What if the translators break this in some other language?

Reply:
Hm, looks like we already do it in a other keys (see `I18N_COMPLETED_STORY` below.)

Reply:
Hey @vojtechjelinek, I don't think I understand what you mean by 'angular markdown things', sorry. If you're referring to the `<[classroomName]>` part of the string which is responsible for variable replacement during runtime, then I'm fairly certain it won't cause any issues as the same format is being used elsewhere for variable replacement in other i18n keys, and the ['adding new translations' wiki page](https://github.com/oppia/oppia/wiki/Adding-new-translations-for-i18n) also gives a similar example for variable replacement.

Please let me know if that answers your question.

Reply:
@kevintab95 Okay, in general, how do we prevent the translators from breaking the tags?

Reply:
Since the actual translations happen outside our platform, what we do is to explicitly tell the translator not to translate anything inside `<...> or <[...]> brackets`. This is mentioned [here](https://translatewiki.net/wiki/Translating:Oppia) in the translatewiki page for Oppia.

Reply:
Is this verified when we merge the translations into the develop branch?

Reply:
(Please ignore if this question was directed towards Kevin) I'm sorry but I'm not sure how we verify that the translators don't accidentally break the tags.

However, I've found past PRs from translatewiki where translations with such tags were added (for example, [line 570 in this file's diff](https://github.com/oppia/oppia/pull/11110/files#diff-b89e30a33bbefad2c0fbc0250cb3dc56f4214ee05241b3411bea8cf17191d764)), so I'm assuming some sort of verification is indeed done.

I've noticed @seanlip merge translatewiki PRs in the past so I have requested him to take a look at this comment.

@seanlip PTAL!

Reply:
Hey @vojtechjelinek, I believe [this function in base_test.py](https://github.com/oppia/oppia/blob/ff0f6f91d65025a1d5123690ba342e3173d97339/core/controllers/base_test.py#L921) is meant for verification of the such translations.